web/admin/src/BigScreen.js

```diff
@@ -172,9 +172,11 @@ const BigScreen = ({ className, fbc, sessionId }) => {
               </div>
               <div className="leaderboard-points">
                 {p.score} {p.score === 1 ? 'pt' : 'pts'}
-                <p className="leaderboard-time">
-                  Avg {p.time ? Math.round(10 * (p.time / p.score / 1000)) / 10 : 0} s
-                </p>
+                {p.score && (
```

Comment:
same question as for `/mobile` with score of `0`.  Does it need to be `p.score > 0 && ...`?